packages/frontend/src/components/ChooseAppAndEventSubstep/index.tsx

```diff
@@ -79,6 +81,13 @@ function ChooseAppAndEventSubstep(
   const apps: IApp[] = data?.getApps?.filter((app: IApp) =>
     isTrigger ? !!app.triggers?.length : !!app.actions?.length,
   )
+
+  apps.sort((a, b) => {
+    if (a.description) {
+      return -1
```

Comment:
if there are multiple apps with descriptions, do we also want to sort alphabetically by name within this group of apps?

Reply:
nvm discussed; we will only have 1 app with description from now